Sources/BraveWallet/Extensions/BraveWalletExtensions.swift

```diff
@@ -309,7 +309,7 @@ extension BraveWallet.BlockchainToken {
   
   var nftTokenTitle: String {
     if isErc721, let tokenId = Int(tokenId.removingHexPrefix, radix: 16) {
-      return "\(name) #\(tokenId)"
+      return "\(symbol) #\(tokenId)"
```

Comment:
on Figma, it's supposed to be `symbol+token id`
before erc721 `metadata.name` does not have any token id information. but now it does. 
I will change the `NFTIconView` to use `metadata.name` instead of this extension.